### PR TITLE
Update

### PR DESCRIPTION
**Pull request by hwangsihu**

### CODE REVIEW

**Comment by infnibor** [comment]
@hwangsihu bad lavalink config

**Comment by hwangsihu** [comment]
Did you mean lyrics plugin?

**Comment by LucasB25** [comment]
> Did you mean lyrics plugin?

You want to add content that is not yet official

**Comment by LucasB25** [comment]
@hwangsihu Remove the content you put in lavalink that has not been made official in lavalink and lavasrc

**Comment by LucasB25** [comment]
image
image

I hope the owner was tested correctly. Because the problem with was when the bot had no permission

File: biome.json
Comment (appujet) [inline biome.json:1]: What is this changes I can't understand why need this? 

Can you explain me why need this ?

**Pull request merged**